rosstatic/launch/example.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+    <node pkg="rosstatic" name="rosstatic" type="main.py" clear_params="true" output="screen">
+        <param name="index" value="$(find aruco_pose)/www/index.html"/>
```

Comment:
I'm a little confused by this, why wouldn't we want to have `rosstatic` build us a list of packages as an example (just like it does when `index` is not set)?

Reply:
Yeah, that's legit, I just wanted to demonstrate with this .launch-file, that such parameter exists.

Reply:
Okay, sounds fair. I've got a couple suggestions, then:
 - How about another .launch file that does not have this parameter (or maybe make this an argument somehow)?
 - Why `aruco_pose` and not something from this package?

Reply:
`aruco_pose` is a typo, I wanted to change it to `my_package` (yeah, this launch-file would fail by default).

Reply:
About first, question, this package doesn't need launch-file, I mean it's useful with default params.